sandbox/engraving/CMakeLists.txt

```diff
@@ -23,24 +23,24 @@ set(CMAKE_MODULE_PATH
 include(SetupBuildEnvironment)
 include(GetPlatformInfo)
 
-set(QT_MIN_VERSION    "5.15.0")
-find_package(Qt5Core ${QT_MIN_VERSION} REQUIRED)
-find_package(Qt5Gui ${QT_MIN_VERSION} REQUIRED)
-#find_package(Qt5Svg ${QT_MIN_VERSION} REQUIRED)
-#find_package(Qt5Widgets ${QT_MIN_VERSION} REQUIRED)
+set(QT_MIN_VERSION "6.0.0")
```

Comment:
Not 6.2.2 here?

Reply:
Hm, forgot about this one. Theoretically, it wouldn't be necessary here because Engraving only uses QtCore which is fine, but for consistency I'll update it. There's no point in using older Qt 6 versions anyway...